shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/FeedingsScreenUi.kt

```diff
@@ -62,7 +67,7 @@ internal fun FeedingsScreenUI(
                 }
 
                 state.feedingsFiltered.isEmpty() -> {
-                    EmptyState()
+                    EmptyState(state.feedingsCategory == FeedingFilterCategory.PENDING)
```

Comment:
We should also check if user has reviewed all feedings in this session and only then show "Well done!"

Reply:
In that case, a feeding reviewed would be moved out to Approved or Rejected, feedings list would be updated and pending feedings is empty. As far as I know "Oops" screen doesn't appear on pending tab never.

Reply:
@ozkarram please check comments in the ticket. We have to check if user reviewed at least 1 feeding.